src/main/java/com/codeborne/selenide/Selectors.java

```diff
@@ -147,6 +148,16 @@ public static By shadowCss(String target, String shadowHost, String... innerShad
     return ByShadow.cssSelector(target, shadowHost, innerShadowHosts);
   }
 
+  /**
+   * @see ByDeepShadow#cssSelector(java.lang.String)
+   * @since v6.8.0
+   */
+  @CheckReturnValue
+  @Nonnull
+  public static By shadowDeepCss(String target) {
```

Comment:
@BorisOsipov  What would be a better name: `shadowDeepCss` or `deepShadowCss`?

Reply:
`shadowDeepCss` seems better because with IDE auto complete you will start search `shadow` first rather than deep, when you want some shadow dom selectors

Reply:
Logish!